app/models/financialDetails/WhatYouOweChargesList.scala

```diff
@@ -49,12 +50,17 @@ case class WhatYouOweChargesList(balanceDetails: BalanceDetails, chargesList: Li
       && overdueChargeList.exists(_.lpiWithDunningLock.getOrElse[BigDecimal](0) > 0)) true
     else false
 
-
   def interestOnOverdueCharges: Boolean =
     if (overdueChargeList.exists(_.interestOutstandingAmount.isDefined)
       && overdueChargeList.exists(_.latePaymentInterestAmount.getOrElse[BigDecimal](0) <= 0)) true
     else false
 
+  def getRelevantDueDate: LocalDate =
```

Comment:
can we wrap this statement into Try block and throw another exception if there is no value of `bcdChargeType`